poc/tests/test_field.py

```diff
@@ -49,6 +49,43 @@ def run_ntt_field_test(self, cls: type[NttField]) -> None:
         # Test generator.
         self.assertTrue(cls.gen()**cls.GEN_ORDER == cls(1))
 
+        for logN in range(1, 8):
+            self.run_nth_roots_test(cls, logN)
+
+        for logN in range(8):
+            self.run_ntt_test(cls, logN)
+            self.run_ntt_star_test(cls, logN)
+
+    def run_nth_roots_test(self, cls: type[NttField], logN: int) -> None:
```

Comment:
Would you mind adding a comment to explain what's going on in this test? Here and below.